core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/util/CamelComponentUtils.java

```diff
@@ -173,7 +183,7 @@ public static boolean isUnsupportedProperty(Parameter p) {
 
 	public static String[] getOneOfList(Parameter p) {
 		String[] choices = p.getOneOf().split(",");
-		ArrayList<String> res = new ArrayList<String>();
+		List<String> res = new ArrayList<>();
```

Comment:
I prefer like that for these reasons:
- shorter lines
- even if it is internal, I think that it is a good habit to take
- my eyes are used to read List instead of ArrayList
- I think that having one more import is not an issue, they are collapsed anyway in the IDE